src/chainstate/stacks/index/storage.rs

```diff
@@ -41,6 +42,7 @@ use std::path::{
 };
 
 use std::os;
+use std::iter::FromIterator;
 
 #[cfg(target_os = "unix")]
```

Comment:
I think we can drop this if we're using `sync_all()`?

Reply:
Addressed in d31f0207